src/PowerShellEditorServices/Services/Analysis/AnalysisService.cs

```diff
@@ -147,7 +146,7 @@ internal static string GetUniqueIdFromDiagnostic(Diagnostic diagnostic)
             EnsureEngineSettingsCurrent();
 
             // Create a cancellation token source that will cancel if we do or if the caller does
```

Comment:
The comment here is no longer true with the `CreateLinked` method removed

Reply:
removed the comment. This is no longer needed because it's not really possible to cancel a `didChange` notification.